src/WalkingModule/app/robots/iCubGazeboV2_5/dcm_walking/joypad_control/robotControl.ini

```diff
@@ -13,6 +13,18 @@ joint_is_stiff_mode     (true, true, true,
                          true, true, true, true, true, true,
                          true, true, true, true, true, true)
 
+joint_damping_gain      (0.1, 0.1, 0.1,
+                         0.1, 0.1, 0.1, 0.1,
+                         0.1, 0.1, 0.1, 0.1,
+                         0.1, 0.1, 0.1, 0.1, 0.1, 0.1,
+                         0.1, 0.1, 0.1, 0.1, 0.1, 0.1)
+
+joint_stiffness_gain     (0.1, 0.1, 0.1,
+                         0.1, 0.1, 0.1, 0.1,
+                         0.1, 0.1, 0.1, 0.1,
+                         0.1, 0.1, 0.1, 0.1, 0.1, 0.1,
+                         0.1, 0.1, 0.1, 0.1, 0.1, 0.1)
+
```

Comment:
Actually here we receive the gains for the compliance mode and it can be a improvement that we are able to set the parameters of the compliance mode in the configuration file.
https://github.com/robotology/walking-controllers/blob/66f272efcdaad8e1fd9952ef78c13b31999e6a9f/src/RobotInterface/src/Helper.cpp#L229